terraform/etl/55-aws-glue-icaseworks_ingest_etl.tf

```
@@ -0,0 +1,35 @@
+module "icaseworks_ingest_to_raw" {
+  source                    = "../modules/aws-glue-job"
+  is_production_environment = local.is_production_environment
+  is_live_environment       = local.is_live_environment
+
+  count = local.is_live_environment ? 0 : 0
+
+  department                     = module.department_housing_data_source
```

I think you want the data and insight department for this so should be
```suggestion
  department                     = module.department_data_and_insight_data_source
```